Classes/Core/Testbase.php

```diff
@@ -625,7 +677,7 @@ public function setUpBasicTypo3Bootstrap($instancePath): ContainerInterface
         // Reset state from a possible previous run
         GeneralUtility::purgeInstances();
 
-        $classLoader = require __DIR__ . '/../../../../autoload.php';
+        $classLoader = require $this->getPackagesPath() . '/autoload.php';
```

Comment:
even if naming of $classLoader is somehow reasonable, I don't like that it's used with the same meaning like $autoloadFile (line 240).  
About the assignment, perhaps a method is better than individual notation for each case?

Reply:
Same as above. The naming the same as before and unrelated to the goal of this change. Can be discussed and tackled in a dedicated change.

I want to stay on goal related stuff for this change (which is how we tend to work in the TYPO3 world). 

Anway, I would disagree with you here.

L680 requires the file and therefor the composer autoloader. L240 containts only the "path" to the autoload file, which is replaced in written files. Two different use-cases, and from my-side the naming are technical correct in both cases.

If you really want to discuss this, open a dedicated issue for this. This is absoluty unrelated to this change.